vllm/entrypoints/openai/api_server.py

```diff
@@ -82,7 +82,8 @@ async def lifespan(app: FastAPI):
     async def _force_log():
         while True:
             await asyncio.sleep(10)
-            await async_engine_client.do_log_stats()
+            with suppress(Exception):
```

Comment:
@joerunde - do we need this? Dont we suppress the warning inside client.py?

Reply:
Ah, yeah you're right. I can clean that up tomorrow